rskj-core/src/main/resources/reference.conf

```diff
@@ -309,9 +309,11 @@ rpc {
     providers {
         web {
             cors = "localhost"
+            max_payload_size = 52428800 # 5mb
             http {
                 enabled = true
                 bind_address = localhost
+                max_batch_requests_size = 100
```

Comment:
it looks like that `max_batch_requests_size` is used by both `Web3HttpServer` and `Web3WebSocketServer`. Shouldn't it be sitting at the `rpc.providers.web` level?

On the other side, it seems that `max_payload_size` plays the same role as `max_frame_size` defined in `rpc.providers.ws`. So I'd rather move `max_payload_size` to the `rpc.providers.http` level and rename it to `max_frame_size` for consistency reason. WDYH?

Reply:
Done

Reply:
@rmoreliovlabs , sorry for misleading.. what I meant here was that `rpc.providers.web.max_payload_size` you added (now its name is `max_frame_size`) is similar to `rpc.providers.web.ws.max_aggregated_frame_size`, so it'd probably make sense to have the one with the same name in `rpc.providers.web.http`?

Reply:
Done